examples/aco_tsp/app.py

```diff
@@ -35,6 +36,8 @@ def circle_portrayal_example(agent):
     },
 }
 
+model = AcoTspModel()
```

Comment:
Normally I try to avoid using `model` as variable name for Model instances, since it can lead to both namespace issues (when importing from `model`) and just general confusion.

I often use `model1` or `<name>_model`.

Reply:
Interesting, never thought of that really. We should have a best practice for this

Reply:
I think I had an multiple hours long debugging session a few years ago, that concluded this. Since then all my instances get an `1` attached unconsciously.

Just now remembered why I did that

Reply:
Marking this as resolved, but maybe something to keep in the back of our heads.